example/hello-populator/deploy.yaml

```diff
@@ -61,11 +61,11 @@ spec:
       serviceAccount: hello-account
       containers:
         - name: hello
-          image: k8s.gcr.io/sig-storage/hello-populator:v0.1.0
+          image: k8s.gcr.io/sig-storage/hello-populator:v1.0.0
           imagePullPolicy: IfNotPresent
           args:
             - --mode=controller
-            - --image-name=k8s.gcr.io/sig-storage/hello-populator:v0.1.0
+            - --image-name=k8s.gcr.io/sig-storage/hello-populator:v1.0.0
```

Comment:
The version might be updated like v1.0.1, and new image would need to be published.